<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4VG4ENC" name="Home" tests="3" failures="1" timestamp="11 Jan 2023 04:30:16 GMT" time="58.378" errors="0">
  <testcase name="launchHomePage_Test" time="1.542" classname="com.automation.ui.digitaldeployer.testcases.index.IndexPageTest"/>
  <testcase name="verifyContinueBeforeLogin" time="10.202" classname="com.automation.ui.digitaldeployer.testcases.login.LoginTest"/>
  <testcase name="launchAddNewWorkspace_Test" time="34.729" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:88)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspace_Test(HomePageTest.java:86)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspace_Test -->
</testsuite> <!-- Home -->
